atlp-rwanda/rca-phantom-team1-bn
# Feature reset password

## What does this PR do

## Description
- This PR does 

## How to manually test this 
- Check in the branch `feature-resetPassword`
- Run the server
- Open the UI/Swagger API documentation to test the reset password functionality as any user.

## Relevant Trello story 
https://trello.com/c/g4q64mw9/22-reset-password

## Screenshots
![image](https://user-images.githubusercontent.com/53254885/236396918-f11015ba-4e4b-4a81-a1fb-71e3622a9aa5.png)
![image](https://user-images.githubusercontent.com/53254885/236396979-c9968a40-4b91-4312-a894-fd3052357ba8.png)
![image](https://user-images.githubusercontent.com/53254885/236698903-d6beaf15-bd33-4e31-84cb-abd4ea4f2d2e.png)
![image](https://user-images.githubusercontent.com/53254885/236698908-814d54eb-8b2a-49b0-b76c-6659303d6ff5.png)
